jenkins/Jenkinsfile.linux

```diff
@@ -332,14 +332,14 @@ pipeline {
                   stage("Publish") {
                     environment {
                       GITHUB_LOGIN = credentials('github-rstudio-jenkins')
-                      DAILIES_PATH = "${PRODUCT}/${OS}-${utils.getArchForOs(env.OS, env.ARCH)}"
+                      DAILIES_PATH = "${PRODUCT}/${OS}-${utils.getArchForOs(OS, ARCH)}"
                     }
 
                     steps {
                       dir("${PACKAGE_DIR}") {
                         script {
                           // publish build to dailies page
-                          utils.publishToDailiesSite PACKAGE_FILE, DAILIES_PATH
+                          utils.publishToDailiesSite PACKAGE_FILE, DAILIES_PATH, AWS_PATH
```

Comment:
I think this needs to happen on `Jenkinsfile.session` once these changes are on the pro branch, right?